src/test/kotlin/com/igorwojda/list/countuniquevalues/challenge.kt

```diff
@@ -4,7 +4,9 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countUniqueValues(list: List<Int>): Int {
-    TODO("not implemented")
+    val set: MutableSet<Int> = mutableSetOf()
+    list.forEach { set.add(it) }
```

Comment:
Remember you have copy constructors in kotlin collections: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/to-set.html